src/advanced/time.md

```diff
@@ -7,6 +7,8 @@ This service allows determining time,
 importing it from the external world to the blockchain
 and keeping its current value in the blockchain.
 
+Time Oracle is supported in [Java][java-time-oracle] as well.
```

Comment:
I'd move the link `[exonum-time]` here and state the supported languages, sth like "Time Oracle is available in Rust[link] and Java[link]/for Rust and Java services".